src/python/README.md

```diff
@@ -8,7 +8,7 @@ Beta : Core behavior well-used and proven; bugs lurk off the beaten path.
 
 PREREQUISITES
 -------------
-- Python 2.7, virtualenv, pip
+- virtualenv, pip, python-dev, python3-dev
```

Comment:
python-dev is definitely needed; python3-dev will be needed soon enough that I figured to include it now.